packages/runtime/container-runtime/src/gc/gcUnreferencedStateTracker.ts

```diff
@@ -25,6 +25,54 @@ class TimerWithNoDefaultTimeout extends Timer {
 	}
 }
 
+/** The collection of UnreferencedStateTrackers for all unreferenced nodes */
+export class UnreferencedNodeTrackers extends Map<string, UnreferencedStateTracker> {
```

Comment:
Do we need to add unit tests for these?

Reply:
N/A - Not worth testing the single feature of calling `stopTracking` on delete.